storage/migrations/16_validator_staking_history.up.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+
+CREATE TABLE chain.validator_balance_history
+(
+    id base64_ed25519_pubkey NOT NULL,
+    epoch UINT63 NOT NULL,
+    PRIMARY KEY (id, epoch),
+    escrow_balance_active UINT_NUMERIC NOT NULL,
+    escrow_balance_debonding UINT_NUMERIC NOT NULL
+);
+CREATE INDEX ix_validator_balance_history_id_epoch ON chain.validator_balance_history (id, epoch);
```

Comment:
Is this redundant since `(id, epoch)` is already a primary key?

Reply:
good catch, removed